licenses.d2iq.yaml

```
@@ -630,3 +631,13 @@ resources:
       - url: https://github.com/kubernetes/kubernetes
         ref: master
         license_path: LICENSE
+  - container_image: docker.io/bitnami/kubectl:1.30.5
```

```suggestion
  - container_image: docker.io/bitnami/kubectl:1.29.6
```
NKP v2.12.0 is based on k8s v1.29.6

---

`kubectl` @ 1.30 is surely backward compatible. The idea was that we tested with these dependencies. I doubt `kubectl` matters though - I'll back it out.